colin-api/src/colin_api/models/filing.py

```diff
@@ -863,6 +863,10 @@ def add_filing(cls, con, filing: Filing) -> int:
                 Business.update_corporation(
                     cursor=cursor, corp_num=corp_num, date=agm_date, annual_report=is_annual_report)
 
+                # Freeze entity for Alteration
+                if filing.filing_type == 'alteration':
+                    Business.update_corp_frozen_type(cursor, corp_num, Business.CorpFrozenTypes.COMPANY.value)
```

Comment:
As discussed with @Sienna-Blumstengel and @thorwolpert , if it is an affiliation we don't need to check the corp type and freeze the corporation.